modules/video_coding/codecs/multiplex/multiplex_encoder_adapter.cc

```diff
@@ -93,7 +93,7 @@ int MultiplexEncoderAdapter::InitEncode(
       key_frame_interval_ = video_codec.H264()->keyFrameInterval;
       video_codec.H264()->keyFrameInterval = 0;
       break;
-#ifndef DISABLE_H265
+#ifdef WEBRTC_USE_H265
```

Comment:
those does not need to be behind macro as well.

Reply:
Removed.